contrib/natss/pkg/dispatcher/dispatcher/dispatcher.go

```diff
@@ -35,28 +36,25 @@ const clientID = "knative-natss-dispatcher"
 type SubscriptionsSupervisor struct {
 	logger *zap.Logger
 
-	receiver   *provisioners.MessageReceiver
-	dispatcher *provisioners.MessageDispatcher
-
-	natssConn *stan.Conn
-
-	subscriptionsMux sync.Mutex
-	subscriptions    map[provisioners.ChannelReference]map[subscriptionReference]*stan.Subscription
+	receiver            *provisioners.MessageReceiver
+	dispatcher          *provisioners.MessageDispatcher
+	connect             chan struct{}
+	natssURL            string
+	subscriptionsMux    sync.Mutex
```

Comment:
Add a comment saying when this has to be held.

Reply:
done